static/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Movies APP</title>
+    <link href="./../static/style.css" rel="stylesheet">
+    <script src="./../static/index.js"></script>
+  </head>
+  <body>
+    <main>
+      <h1 class="site__title">Movies APP</h1>
+
+      <header class="movie__wrapper">
+        <h2 class="movie__title">The Matrix</h1>
+        <div class="movie__description">
+        <p>"The Matrix," directed by the Wachowskis, is a 1999 science fiction film that explores themes of reality, freedom, and control. The story follows Thomas Anderson (Keanu Reeves), a computer hacker known by the alias "Neo." Neo is disillusioned with his mundane life and haunted by mysterious online encounters with a figure named Morpheus (Laurence Fishburne), who reveals a shocking truth: the world as Neo knows it is a simulated reality called the Matrix.</p>
+
+<p>Morpheus explains that in reality, intelligent machines have enslaved humanity, using their bodies as an energy source while keeping their minds imprisoned in the Matrix. Neo learns he is "The One," prophesied to liberate humanity from the machines. Morpheus and his crew, including the skilled fighter Trinity (Carrie-Anne Moss), train Neo to manipulate the Matrix's code, giving him superhuman abilities within the simulation.</p>
+
+<p>The group battles against the Matrix's sentient security programs, known as Agents, led by the relentless Agent Smith (Hugo Weaving). As Neo's powers grow, he confronts his destiny, culminating in a climactic showdown with Agent Smith. Neo ultimately embraces his role as humanity's savior, heralding the beginning of the fight to free humanity from the Matrix's grip. The film ends with Neo's promise to expose the truth and lead the resistance against the machines.</p></div>
+        <img class="movie__poster" src="./../assets/matrix.png" alt="IMDb Logo" />
```

Comment:
Sto dobivas s ovim da ti je img zadnji i onda settas order kroz css? Zasto ne samo staviti img tag gore? Sto ne kuzim?

Reply:
Iz semantičke i accesibility perspektive bolje je imati heading kao prvi element

Reply:
Ne kazem da nisi u pravu, ali hejtam to.

Reply:
Don't hate the player, hate the game 🤣